index.html

```diff
@@ -0,0 +1,453 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>aaaaaaaaaaaaaaaaaaa</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">sadsfdg Mejri</h1>
```

Comment:
The size of the logo text on the Figma design is 18px, while here it is 60px. Kindly fix the size and also provide a valid title - the person's name whose portfolio is being built.